CHANGELOG.md

```diff
@@ -4,6 +4,42 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.3](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.3) - 2021-12-17
+
+Zebra's latest beta works towards enforcing all consensus rules by validating JoinSplit Groth16 proofs
```

Comment:
Are we validating all consensus rules with this release? If so, we should be explicit about it.

Otherwise, what's missing? Should we explicitly say what we need to do for full validation?

Reply:
I see that we need to fix the Sprout validation now.